### PR TITLE
fix: remove cookie policy banner from authn

### PR DESCRIPTION
**Pull request by attiyaIshaque**
### Description

The frontend-app-authn is currently the only repo depending on the frontend-component-cookie-policy-banner. We are not using this dependency on the prod or stage anymore. This PR removes the dependency.

#### JIRA
https://2u-internal.atlassian.net/browse/VAN-1414

### CODE REVIEW

File: src/base-component/BaseComponent.jsx [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import React from 'react';
 
-import CookiePolicyBanner from '@edx/frontend-component-cookie-policy-banner';
-import { getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
-import { getLocale } from '@edx/frontend-platform/i18n';
```
Comment (shahbaz-arbisoft) [inline src/base-component/BaseComponent.jsx:6]: Is this method solely intended for the cookie policy banner or is it used for other purposes as well?

File: src/base-component/BaseComponent.jsx [hunk 2]
```diff
@@ -1,9 +1,6 @@
 import React from 'react';
 
-import CookiePolicyBanner from '@edx/frontend-component-cookie-policy-banner';
-import { getConfig } from '@edx/frontend-platform';
```
Comment (shahbaz-arbisoft) [inline src/base-component/BaseComponent.jsx:4]: Is this method solely intended for the cookie policy banner or is it used for other purposes as well?

File: src/base-component/BaseComponent.jsx (hunk 1 above)
Comment (attiyaIshaque) [reply src/base-component/BaseComponent.jsx:6]: It's only used for cookie policy banner.

**Comment by shahbaz-arbisoft** [comment]
Everything appears satisfactory to me.

**Review by shahbaz-arbisoft — APPROVED** [review]

**Pull request merged**